src/store/activities/actions.js

```diff
@@ -1,9 +1,29 @@
 import database from 'firebase/firebase';
+import { createIntl, createIntlCache } from 'react-intl';
+import flatten from 'flat';
 
 import { generateActivitiesItemsPath } from 'helpers/paths';
+import { errorNotification, successNotification } from 'components/elements';
+import messagesPL from 'translations/pl.json';
+import messagesEN from 'translations/en.json';
 
 import { ADD_ACTIVITY, REMOVE_ACTIVITY, EDIT_ACTIVITY, SET_ACTIVITIES } from './actionTypes';
 
+const messages = {
+  pl: flatten(messagesPL),
+  en: flatten(messagesEN),
+};
+
+const language = navigator.language.split(/[-_]/)[0];
+
+const intl = createIntl(
```

Comment:
for store, you could make it more reusable by creation intl in separate file and use only exported intl variable here.

Reply:
even go further and export function which receive path to translation (similar as describe hook for components)